config/daqsystemtest/moduleconfs.data.xml

```diff
@@ -571,6 +589,14 @@
  <attr name="expected_timestamp_step" type="s32" val="1000"/>
 </obj>
 
+<obj class="PDSRawDataProcessor" id="PDSRawDataProc">
```

Comment:
probably should follow naming conventions for other ids. something like `def-pds-processor`